proxy/handler.go

```diff
@@ -0,0 +1,52 @@
+package proxy
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/ONSdigital/dp-legacy-cache-proxy/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+func (proxy *Proxy) Handler(ctx context.Context, w http.ResponseWriter, req *http.Request, cfg *config.Config) {
```

Comment:
Another thing is that this file name and this method name should probably be changed to something else, since this is not really a handler, and it does not return a handler either. I'm not sure what the right name could be... So far, this is a method that passes the request to Babbage, but later on this will be the central place while the whole proxy logic occurs.